toolchain/check/context.cpp

```diff
@@ -327,8 +328,11 @@ auto Context::LookupUnqualifiedName(Parse::NodeId node_id,
             .inst_id = lexical_result};
   }
 
-  // We didn't find anything at all.
-  DiagnoseNameNotFound(node_id, name_id);
+  if (required) {
+    // We didn't find anything at all.
```

Comment:
```suggestion
  // We didn't find anything at all.
  if (required) {
```

I think the comment is explaining the current state, regardless of the value of `required`, so probably should remain outside the `if`.